src/index.ts

```diff
@@ -47,82 +75,96 @@ export interface IDKeys {
   ipnsKey: Key
 }
 
+export interface CreateOptions {
+  v1Compatible?: boolean
+}
+
+const defaultCreateOptions: CreateOptions = {
+  v1Compatible: true
+}
+
 /**
- * Creates a new ipns entry and signs it with the given private key.
- * The ipns entry validity should follow the [RFC3339]{@link https://www.ietf.org/rfc/rfc3339.txt} with nanoseconds precision.
+ * Creates a new IPNS record and signs it with the given private key.
+ * The IPNS Record validity should follow the [RFC3339]{@link https://www.ietf.org/rfc/rfc3339.txt} with nanoseconds precision.
  * Note: This function does not embed the public key. If you want to do that, use `EmbedPublicKey`.
  *
  * @param {PeerId} peerId - peer id containing private key for signing the record.
- * @param {Uint8Array} value - value to be stored in the record.
+ * @param {string} value - value to be stored in the record.
  * @param {number | bigint} seq - number representing the current version of the record.
  * @param {number} lifetime - lifetime of the record (in milliseconds).
+ * @param {CreateOptions} options - additional create options.
  */
-export const create = async (peerId: PeerId, value: Uint8Array, seq: number | bigint, lifetime: number): Promise<IPNSEntry> => {
+export const create = async (peerId: PeerId, value: string | Uint8Array, seq: number | bigint, lifetime: number, options: CreateOptions = defaultCreateOptions): Promise<IPNSRecord> => {
```

Comment:
@achingbrain the `value` must be a content path and a string is the most natural way of representing it: https://specs.ipfs.tech/ipns/ipns-record/#ipns-record. We are keeping `Uint8Array` because for some reason `js-ipns` was allowing serialized CIDs in the `value` field. See https://github.com/ipfs/js-ipns/pull/234#discussion_r1289202200